mlflow/metrics/genai/model_utils.py

```diff
@@ -76,32 +77,29 @@ def _call_openai_api(openai_uri, payload):
 
     import openai
 
-    from mlflow.openai.api_request_parallel_processor import process_api_requests
     from mlflow.openai.utils import _OAITokenHolder
 
     api_token = _OAITokenHolder(os.environ.get("OPENAI_API_TYPE", "openai"))
 
-    resp = process_api_requests(
-        [openai_provider._add_model_to_payload_if_necessary(payload)],
-        openai.ChatCompletion,
-        api_token=api_token,
-        max_requests_per_minute=3_500,
-        max_tokens_per_minute=90_000,
-    )[0]
-
-    if "error" in resp:
-        error_type = resp["error"]["type"]
-        if error_type == "invalid_request_error":
-            raise MlflowException(
-                f"Invalid Request to OpenAI. Error response:\n {resp}", error_code=BAD_REQUEST
-            )
-        elif error_type == "authentication_error":
-            raise MlflowException(
-                f"Authentication Error for OpenAI. Error response:\n {resp}",
-                error_code=UNAUTHENTICATED,
-            )
-        else:
-            raise MlflowException(f"Error response from OpenAI:\n {resp}")
+    try:
+        resp = process_one_api_request(
+            openai_provider._add_model_to_payload_if_necessary(payload),
+            openai.ChatCompletion,
+            api_token=api_token,
+            max_requests_per_minute=3_500,
+            max_tokens_per_minute=90_000,
```

Comment:
Why do we need this? Shouldn't this be only used for bulk usecase? The parallel processing is already handled here in a separate process.